models/users.js

```diff
@@ -84,6 +84,39 @@ const getJoinData = async (userId) => {
   }
 };
 
+/**
+ * Fetches users with the given skill
+ *
+ * @param skill { string }: Skill
+ * @return @return {Promise<users>}
+ */
+
+const getSuggestedUsers = async (skill) => {
+  try {
+    const data = await itemModel.where("itemtype", "==", "USER").where("tagid", "==", skill).get();
+    const usersId = [];
+    let users = [];
+
+    const dataSet = new Set();
+
+    if (!data.empty) {
+      data.forEach((doc) => {
+        if (!dataSet.has(doc.data().itemid)) {
+          dataSet.add(doc.data().itemid);
+          usersId.push(doc.data().itemid);
+        }
+      });
+      const usersArray = usersId.map((userId) => fetchUser({ userId }));
+      users = await Promise.all(usersArray);
+    }
+
+    return { users };
+  } catch (err) {
+    logger.error("Error in getting suggeted", err);
```

Comment:
Typo

Reply:
@vinayak-trivedi Please fix this in a future feature :D

Reply:
fixed it, thanks